QgisModelBaker/gui/generate_project.py

```diff
@@ -190,10 +191,37 @@ def accepted(self):
         configuration.dbschema = configuration.dbschema or configuration.database
         self.save_configuration(configuration)
 
-        # create schema with superuser
         db_factory = self.db_simple_factory.create_factory(db_id)
-        res, message = db_factory.pre_generate_project(configuration)
 
+        try:
+            # raise warning when the schema or the database file already exists
+            config_manager = db_factory.get_db_command_config_manager(configuration)
+            db_connector = db_factory.get_db_connector(
+                config_manager.get_uri(configuration.db_use_super_login) or config_manager.get_uri(), configuration.dbschema)
+
+            if db_connector.db_or_schema_exists():
+                if interlis_mode:
+                    warning_box = QMessageBox(self)
+                    warning_box.setIcon(QMessageBox.Warning)
+                    warning_title = self.tr("{} already exists").format(
+                        db_factory.get_specific_messages()['db_or_schema']
+                    ).capitalize()
+                    warning_box.setWindowTitle(warning_title)
+                    warning_box.setText(self.tr("{db_or_schema}: {db_or_schema_name}\n\nDo you want to "
```

Comment:
```suggestion
                    warning_box.setText(self.tr("{db_or_schema}: {db_or_schema_name} already exists.\n\nDo you want to "
```

Reply:
I would prefer: 
```
{db_or_schema} already exists: 
{db_or_schema_name}

Do you want to import anyway?
```
Because {db_or_schema_name} can be quite long when it's a path to the gpkg.
But then it's the same like the title on the top line, and that's not so nice.

Reply:
Agreed on the long filename.

How the title is displayed depends on the platform, some systems may decide to show the title bar somewhere else than directly around the message box. I would put all the information into the dialog, even if it's duplicate.